src/ui/components/panels/loadout/active_loadout/dialogs/downtime/_GetFocused.vue

```diff
@@ -47,12 +47,16 @@ export default Vue.extend({
   },
   methods: {
     addSkill() {
-      this.pilot.Reserves.push(
+      this.pilot.AddReserve(
         new Reserve({
-          id: 'reserve_skill',
+          id: 'reserve_skillfocus',
           type: 'Bonus',
           name: 'Skill Focus',
           description: 'Added via the "Get Focused" Downtime Action',
+          bonuses: [{
+            "id": "skill_point",
+            "val": 1
+          }],
```

Comment:
Following the pattern in [lancer-data/reserves.json](https://github.com/massif-press/lancer-data/blob/master/lib/reserves.json#L1-L11)